repose-aggregator/tests/performance-tests/src/performanceTest/scala/filters/openapivalidator/OpenApiValidatorFilterSimulation.scala

```diff
@@ -126,4 +150,10 @@ object OpenApiValidatorFilterSimulation {
     ("/pets", StatusCodeMethodNotAllowed),
     ("/pets/17362", StatusCodeOk),
     ("/notPets", StatusCodeNotFound))
+
+  //(role: String   , respcode: Int)
+  private val ScenariosRole = Array[(String, Int)](
+    ("foo", StatusCodeNoContent),
+    ("bar", StatusCodeNoContent),
+    ("baz", StatusCodeUnauthorized))
```

Comment:
I know this was done this way for consistency, but honestly, this should probably just be a map.